lava-v2-jobs-from-api.py

```diff
@@ -106,13 +106,18 @@ def get_job_params(config, test_config, defconfig, opts, build, plan):
     # hack for arm64 dtbs in subfolders
     if arch == 'arm64' and dtb:
         dtb = opts['dtb'] = os.path.basename(dtb)
-
-    job_name = '-'.join([
-        config.get('tree'), config.get('branch'), config.get('describe'),
-        arch, defconfig, dtb or 'no-dtb', device_type.name, plan])
-
-    url_px = '/'.join([
-        build['job'], build['git_branch'], build['kernel'], arch, defconfig])
+    if 'file_server_resource' in build:
+        job_name_prefix = build['file_server_resource'].replace('/', '-')
```

Comment:
Are we sure `'/'` is the only character we need to remove from the file server resource field?  I guess we know it's fine because we're making this value in `build.py` but there's nothing in principle preventing it from having other potentially unwanted characters.  It might be safer to have a function to replace any character that's not in `[a-z0-9-]` with a `-`.  How likely is this going to be an issue?

Reply:
Deemed OK in the context of the current usage of this script within KernelCI.